src/internal_events/exec.rs

```diff
@@ -0,0 +1,51 @@
+use super::InternalEvent;
```

Comment:
In addition to these metrics, I think we may want to publish metrics for:

* `events_in_total`, which was a standard metric added recently in https://github.com/timberio/vector/pull/6758
* `command_executed_total` which would be the total number of times the command was executed
* `command_execution_duration_ns` would would be a histogram of execution times

Reply:
Added